portal/src/components/CodeBlock/fremtindTheme.ts

```diff
@@ -0,0 +1,171 @@
+export default {
+    'code[class*="language-"]': {
+        color: "black",
+        background: "none",
+        textShadow: "0 1px white",
+        fontFamily: "Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace",
+        textAlign: "left",
+        whiteSpace: "pre",
+        wordSpacing: "normal",
+        wordBreak: "normal",
+        wordWrap: "normal",
+        lineHeight: "1.5",
+        MozTabSize: "4",
+        OTabSize: "4",
+        tabSize: "4",
+        WebkitHyphens: "none",
+        MozHyphens: "none",
+        msHyphens: "none",
+        hyphens: "none",
+    },
+    'pre[class*="language-"]': {
+        lineHeight: "1.4",
+    },
+    'pre[class*="language-"]::-moz-selection': {
+        textShadow: "none",
+        background: "#b3d4fc",
```

Comment:
Går det inte och få in färgerna som variabler istället?

Reply:
tenker dette nok er helt greit for bruksområdet. Dette er nok ikke farger som skal gjenbrukes noen steder, så faktisk kan det være mot sin hensikt å lagre de som variabler.